test/extensions/filters/network/common/fuzz/uber_per_readfilter.cc

```diff
@@ -18,27 +18,23 @@ namespace {
 static const int SecondsPerDay = 86400;
 } // namespace
 std::vector<absl::string_view> UberFilterFuzzer::filterNames() {
-  // These filters have already been covered by this fuzzer.
-  // Will extend to cover other network filters one by one.
+  // Add filters that are in the process of being or are robust against untrusted downstream
```

Comment:
Is there some structural way we can ensure that when we change security robustness status that we also update this list? I'm thinking something as simple as a bazel query on network filters by posture and a comparison of count with something we can grep out of this file, or whatever you reckon makes sense here.

Reply:
Done. It is a course check (numbers equal) when generating the extension database.
Matching names would be a bit tougher, because they would match on the canonical filter name vs the extension name